scripts/install_submodules.sh

```diff
@@ -24,46 +24,9 @@ USAGE
   exit 1
 }
 
-clone_project()
-{
-  if [ -z "$2" ];
-  then
-    printf "Empty paths are dangerous in use. Use another path instead for submodules installation or update.\n"
-    exit 1
-  fi
-
-  if [[ ! -d "$2" || ${update} == 1 ]];
-  then
-    if (( update == 1 ));
-    then
-      printf "Remove submodule %s from %s\n" "$1" "$2"
-      rm -rf "$2"
-    fi
-
-    if [ -n "$4" ];
-    then
-      printf "Clone submodule %s (%s %s) into %s\n" "$1" "$3" "$4" "$2"
-    else
-      printf "Clone submodule %s (%s) into %s\n" "$1" "$3" "$2"
-    fi
-    git clone "$1" --branch "$3" --single-branch "$2" --recursive
-    if [ -n "$4" ];
-    then
-      cd "$2" && git checkout "$4"
-    fi
-  else
-    printf "You can update %s manually. Use this script with \"--update\" option.\n" "$2"
-  fi
-}
-
-update=0
-
 while [ "$1" != "" ];
 do
   case $1 in
```

Comment:
Leave it here, but set var `update` by value `--update`. It is needed for common arguments validation to prevent validation logic in callable child scripts